docs/source/conf.py

```diff
@@ -69,10 +68,11 @@
 #
 # get the latest git tag with  the latest commit
 # eg pyro-0.1.2-g4c52460
+# this only matters if you host manually
 version = re.sub('^v', '', os.popen('git describe --tags').read().strip())
 
 # release version
-release = __version__  # eg pyro 0.1.2
+release = version
```

Comment:
Wrap this in `try`/`except` to handle weird git failures?
```
try:
    version = re.sub(...)
except ... :
    version = __version__
```

Reply:
i removed the pyro import up top so the `except ... version = __version__` will throw an exception. in any case these fields are ignored by rtd where we're migrating the docs